static/styles/zulip.css

```diff
@@ -1395,7 +1412,7 @@ td.pointer {
     align-items: center;
     justify-content: space-between;
     border-right: 1px solid hsl(0, 0%, 88%);
-    background-color: hsl(0, 0%, 88%);
+    background-color: hsl(104.2, 100%, 92.5%);
```

Comment:
Temporary color -- the actual recipient header is being worked on in a separate PR.